dotnet/samples/GettingStartedWithProcesses/README.md

```diff
@@ -165,6 +165,27 @@ graph TD
     SingleOrderReadyEvent-->PackFoodStep --> OrderPackedEvent
 ```
 
+### Step04_AgentOrchestration
```

Comment:
don't forget to add `Step04_AgentOrchestration` to the table under `##Examples`

Reply:
Added!